src/LayoutManager/LayoutManager.ts

```diff
@@ -55,24 +55,31 @@ export class LayoutManager {
     this.onBeforeLayout(strictContext);
 
     const layoutResult = this.getLayoutResult(strictContext);
-    layoutResult && this.commitLayout(strictContext, layoutResult);
+    if (layoutResult) {
+      this.commitLayout(strictContext, layoutResult);
+    }
 
     this.onAfterLayout(strictContext, layoutResult);
     this._prevLayoutStrategy = strictContext.strategy;
   }
 
   /**
-   * subscribe to object layout triggers
+   * Attach handlers for events that we know will invalidate the layout when
+   * performed on child objects ( general transforms ).
+   * Returns the disposers for later unsubscribing and cleanup
+   * @param {FabricObject} childObject
+   * @param {RegistrationContext & Partial<StrictLayoutContext>} context
+   * @returns {VoidFunction[]} disposers remove the handlers
    */
-  protected subscribe(
-    object: FabricObject,
+  protected attachHandlers(
+    childObject: FabricObject,
```

Comment:
I think childObject is misleading because it can be the active selection, right? eventTarget perhaps?